src/test/unit/profile-data.test.js

```diff
@@ -884,3 +885,72 @@ describe('getSamplesSelectedStates', function() {
     expect(comparator(2, 0)).toBeGreaterThan(0);
   });
 });
+
+describe('extractProfileFilterPageData', function() {
+  const firstInnerWindowID = 1;
+  const secondInnerWindowID = 2;
+  const thirdInnerWindowID = 3;
+  const fourthInnerWindowID = 4;
```

Comment:
nit: what about using something recalling the URLs in this variable names? Eg `firstInnerWindowID` -> `mozillaInnerWindowId` or `innerWindowIdForMozilla`.

Or even with an object:
```js
const innerWindowIds = {
  mozilla: 1,
  aboutBlank: 2,
  profiler: 3,
  exampleComFrame: 4,
};
```
Then it's easier to follow in the tests.